src/goal_sequence_visualizer.py

```diff
@@ -58,4 +59,4 @@ def init_goal_sequence_visualizer(mat):
 		pub, pub_txt, rate, pose_array, marker_array =	init_goal_sequence_visualizer(server.mat)
 		pub.publish(pose_array)
 		pub_txt.publish(marker_array)
-		rate.sleep()
\ No newline at end of file
+		rate.sleep()
```

Comment:
I will not try to remove the newline at the end, maybe in the future, there might be a problem with us for the version control